File: 1672.richest-customer-wealth.java
/*
 * @lc app=leetcode id=1672 lang=java
 *
 * [1672] Richest Customer Wealth
 */

// @lc code=start
class Solution {
    public int maximumWealth(int[][] accounts) {
        
        int maxWealth = 0;

        for(int i = 0; i < accounts.length; i++){
            int countMoney = 0;
            for(int j = 0; j < accounts[0].length; j++){
                countMoney += accounts[i][j];
            }
            maxWealth = Math.max(maxWealth, countMoney);
        }

        return maxWealth;
    }
}
// @lc code=end

File: 1672.richest-customer-wealth.js
/*
 * @lc app=leetcode id=1672 lang=javascript
 *
 * [1672] Richest Customer Wealth
 */

// @lc code=start
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    
    let wealthiest = 0;

    for(let i = 0; i < accounts.length; i++){

        let countMoney = 0;
        
        accounts[i].forEach((money)=>{
             countMoney += money
        })

        wealthiest = Math.max(countMoney, wealthiest);
    }

    return wealthiest;
};
// @lc code=end

